Content.Server/Items/ItemComponent.cs

```diff
@@ -44,6 +44,7 @@ protected override void GetData(IEntity user, ItemComponent component, VerbData
                 }
 
                 data.Text = Loc.GetString("pick-up-verb-get-data-text");
+                data.IconTexture = "/Textures/Interface/up_arrow.svg.192dpi.png";
```

Comment:
This is re-using the up-arrow icon from the changelog. It's actually a bit too small, relative to the other context menu icons. Eventually this should maybe be replaced by a bigger up-arrow, a hand, or something else?

Reply:
I've now replaced this with a resized pickup arrow, and moved the up arrow icon back into the changelog folder. It now looks like this:
![pickup](https://user-images.githubusercontent.com/60421075/130207207-523ad19f-2599-477b-996b-f12978b67aae.png)